.github/actions/stale_issues/action.yml

```diff
@@ -0,0 +1,49 @@
+name: Filter stale issues
+inputs:
+  token:
+    description: 'The GITHUB_TOKEN secret'
+    required: true
```

Comment:
Doesn't this need a type?

Reply:
Not sure. I was trying to find if they have a specific type for secrets but couldn't find it. I'm afraid that it is usually passing it as a string, although checking the action logs, the token is not shown in the plain.